documentation/src/docs/wire-protocol.md

```diff
@@ -9,104 +9,105 @@ You may obtain a copy of the License at
 -->
 # Streaming Service Wire Protocol
 
-This page describes the proposed Wire Protocol for the Streaming Service. See parent page for description of the service as a whole.
+This page describes the proposed Wire Protocol for the Streaming Service. See [Pravega Concepts](http://pravega.io/docs/latest/pravega-concepts) for more information.
 
-Protocol
-========
+# Protocol
 
-Data is sent over the wire in self-contained "messages" that are either "requests" (messages from the client to the server) or "replies" (which are in response to a request and go back to the client).
+Data is sent over the wire in self-contained "messages" that are either "requests" (messages sent from the client to the server) or "replies" (responses sent from the server to the client).
 
-All requests and replies have an 8 byte header with two fields, (All data is written in BigEndian format):
-1.  Message Type - An integer (4 bytes) identifies the message type and this determines what fields will follow. (Note the protocol can be extended by adding new types)
-2.  Length - Unsigned integer 4 bytes (Messages should be &lt; 2^24, but the upper bits remain zero). How many bytes of data from this point forward are part of this message. (Possibly zero, indicating there is no data)
+All the requests and replies have 8 byte header with two fields. (All data is written in **BigEndian** format).
 
-The remainder of the fields are specific to the type of message. A few important messages are listed below.
+- **Message Type**: A 4 byte Integer identifies the message type and determines the subsequent fields. (Note that the protocol can be extended by adding new types.)
+- **Length**:  A 4 byte Unsigned Integer (Messages should be less than 2^<sup>24</sup>, but the upper bits remain zero). The remaining bits followed by the upper bit are part of the message. (Possibly zero, indicating there is no data).
```

Comment:
Remove this sentence "The remaining bits followed by the upper bit are part of the message. " That is not accurate.
At the beginning: "A 4 byte Unsigned Integer" It doesn't matter if it is signed or not. The value is less than 2^24 and it is always zero or positive. So it makes no difference if it is treated as signed or unsigned.

Reply:
Accepted suggestion